programs/vote/benches/vote_instructions.rs

```diff
@@ -0,0 +1,194 @@
+            (sysvar::clock::id(), clock_account),
+            (authorized_voter_pubkey, AccountSharedData::default()),
+        ];
+        let instruction_accounts = vec![
+            AccountMeta {
+                pubkey: vote_pubkey,
+                is_signer: true,
+                is_writable: true,
+            },
+            AccountMeta {
+                pubkey: sysvar::clock::id(),
+                is_signer: false,
+                is_writable: false,
+            },
+        ];
+        Self {
+            instruction_data,
+            transaction_accounts,
+            instruction_accounts,
+        }
+    }
+
+    fn run(&self) {
+        let _accounts = process_instruction(
+            &self.instruction_data,
+            self.transaction_accounts.clone(),
+            self.instruction_accounts.clone(),
+            Ok(()),
+        );
+    }
+}
+
+struct BenchInitializeAccount {
+    vote_pubkey: Pubkey,
+    vote_account: AccountSharedData,
+    node_pubkey: Pubkey,
+    node_account: AccountSharedData,
+    instruction_data: Vec<u8>,
+    instruction_accounts: Vec<AccountMeta>,
+}
+
+impl BenchInitializeAccount {
+    fn new() -> Self {
+        let vote_pubkey = solana_pubkey::new_rand();
+        let vote_account = AccountSharedData::new(100, VoteState::size_of(), &id());
+        let node_pubkey = solana_pubkey::new_rand();
+        let node_account = AccountSharedData::default();
+        let instruction_data = serialize(&VoteInstruction::InitializeAccount(VoteInit {
+            node_pubkey,
+            authorized_voter: vote_pubkey,
+            authorized_withdrawer: vote_pubkey,
+            commission: 0,
+        }))
+        .unwrap();
+        let instruction_accounts = vec![
+            AccountMeta {
+                pubkey: vote_pubkey,
+                is_signer: false,
+                is_writable: true,
+            },
+            AccountMeta {
+                pubkey: sysvar::rent::id(),
+                is_signer: false,
+                is_writable: false,
+            },
+            AccountMeta {
+                pubkey: sysvar::clock::id(),
+                is_signer: false,
+                is_writable: false,
+            },
+            AccountMeta {
+                pubkey: node_pubkey,
+                is_signer: true,
+                is_writable: false,
+            },
+        ];
+        Self {
+            vote_pubkey,
+            vote_account,
+            node_pubkey,
+            node_account,
+            instruction_data,
+            instruction_accounts,
+        }
+    }
+    pub fn run(&self) {
+        let _accounts = process_instruction(
+            &self.instruction_data,
+            vec![
+                (self.vote_pubkey, self.vote_account.clone()),
+                (sysvar::rent::id(), create_default_rent_account()),
+                (sysvar::clock::id(), create_default_clock_account()),
+                (self.node_pubkey, self.node_account.clone()),
+            ],
```

Comment:
nit: format wise, allocation can be moved to `new()` then call `self.transaction_accounts.clone()` here, same as `BenchAuthorize` does.